packages/driver/src/cypress/stack_utils.ts

```diff
@@ -60,6 +60,58 @@ const stackWithLinesRemoved = (stack, cb) => {
   return unsplitStack(messageLines, remainingStackLines)
 }
 
+const stackWithWrappingLinesRemoved = (stack, specWindow) => {
+  const modifiedStack = stackWithLinesRemoved(stack, (lines) => {
+    // Guard against Cypress being undefined/null (can happen when users quickly reload tests)
+    if (!specWindow?.Cypress) {
+      return lines
+    }
+
+    if (specWindow.Cypress.isBrowser('chrome')) {
+    // There are cases where there are other lines in the stack trace before the invocation (eg. `context.it.only`, `createRunnable`, etc)
+    // Remove lines from the start until the top line starts with 'at eval' or 'at Suite.eval' so that we only keep the actual invocation line.
+      while (
+        lines.length > 0 &&
+      !(
+        lines[0].trim().startsWith('at eval') ||
+        lines[0].trim().startsWith('at Suite.eval')
+      )
+      ) {
+        lines.shift()
+      }
+    } else if (specWindow.Cypress.isBrowser('firefox')) {
```

Comment:
Similarily, this needs to be any firefox (e.g. firefox beta, firefox nightly, etc):

```suggestion
    } else if (specWindow.Cypress.isBrowser({ family: 'firefox' })) {
```